packages/live-preview-sdk/src/__tests__/react.spec.tsx

```diff
@@ -409,7 +414,7 @@ describe('useContentfulInspectorMode', () => {
     });
 
     it('should return null because the inspector mode is disabled', () => {
-      const { result } = renderHook((data) => useContentfulInspectorMode(data), {
+      const { result } = renderHook((data) => useContentfulInspectorMode<any>(data), {
```

Comment:
this any seems like another issue? 🤔

Reply:
I think it's fine? Because otherwise it complains that we don't pass certain fields in
```
result.current({
  locale,
  fieldId: 'title',
}),
```

e.g. we don't pass `entryId` but pass it `initialProps`

Reply:
![Screenshot 2024-05-23 at 15 22 16](https://github.com/contentful/live-preview/assets/3918488/a3ddce49-ee56-4902-a680-f5838d9d3f68)

if we pass entryId on the initialProps it fails on the second call as it expect it to be provided there too